src/main/app/src/app/shared/material-form-builder/form-components/documenten-lijst/documenten-lijst.component.ts

```diff
@@ -54,7 +59,26 @@ export class DocumentenLijstComponent extends FormComponent implements OnInit {
     change($event: MatCheckboxChange, document): void {
         if ($event) {
             this.selection.toggle(document);
-            this.data.formControl.setValue(this.selection.selected.map(value => value.uuid).join(';'));
+            this.setValue();
         }
     }
+
+    setOndertekend($event: MatCheckboxChange, document): void {
```

Comment:
Als ik dit zie denk ik kan dat niet scherper? Iets met checked op de control. Want het is nu dus mogelijk dat de checkbox-control en dit veld niet met elkaar overeenkomen. En hoe vaak je ook klikt, dat komt dan niet meer vanzelf goed.